dbee/clients/redshift_test.go

```diff
@@ -0,0 +1,477 @@
+package clients
```

Comment:
Note for future self: I'd like to add a more common way of mocking connections and rows in the future. Also integration tests would be nice :)

Reply:
[d91671b](https://github.com/kndndrj/nvim-dbee/pull/34/commits/d91671b66fbae0fb20aeeb25f994c4def457d9fd)

I agree. I removed the tests now and will probably add them back in the future once it is easier to mock out connections/rows from the common package 👍